pkgs/by-name/cl/cloudquery/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "cloudquery";
+  version = "6.11.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudquery";
+    repo = "cloudquery";
+    rev = "cli-v${version}";
+    hash = "sha256-zVzHPAN/UAVQF56qcoIFOqtX/Aek3+1lO2O965UYxPM=";
+  };
+
+  vendorHash = "sha256-9YvsVflHr4QSVCYCcWg1Okmncc1sNteMJEAdz92BIgI=";
+
+  CGO_ENABLED = 0;
+
+  modRoot = "cli";
+
+  # Some checks fail to download deps
+  doCheck = false;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/cloudquery/cloudquery/cli/cmd.Version=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/$pname
+  '';
+
+  meta = with lib; {
```

See https://github.com/NixOS/nixpkgs/issues/208242
```suggestion
  meta = {
```

---

Fixed. Thanks for the link. I didn't understand why this was wrong, since it looks like the majority of the packages under `by-name` are currently using `with lib` in the `meta` section.